### DISCUSSION TITLE
6GB HDD Image Download

### DISCUSSION THREAD

**Discussion by sussygamedeveloper** [copy/v86#650] [General]
a empty 6gb .img file so people without access to a (real) computer can have a hard drive. 
im posting this because some people only have chromebooks, and don't have access to qemu to create the file.
https://drive.google.com/file/d/1OIyKvbQhHei7k4vRNCw4yM0Gbd7EaPtI/view?usp=sharing

**Comment by sussygamedeveloper** [comment]
something will definitely go wrong because im a fucking idiot.

**Comment by hello-smile6** [comment]
For creating sparse images on ChromeOS, you can actually use crosh.
1. Press <kbd>Ctrl+Alt+T</kbd>
2. ```bash
    vmc create-extra-disk ~/MyFiles/diskname.img --size 6G
    ```
3. Use your 6GB disk image.

If that doesn't work, type `help vmc` and figure it out from the documentation; I might have the wrong command.

**Reply by sussygamedeveloper** [reply]
oh really! thanks for telling me.

**Reply by sussygamedeveloper** [reply]
well most people happen to be on school chromebooks, and crosh (in most cases) is blocked.
i mean maybe not the whole crosh but DEFINITELY vmc, because virtual machines are disabled on school chromebooks by default.

**Reply by hello-smile6** [reply]
@sussygamedeveloper `vmc create-extra-disk` works even when crostini is disabled.

**Reply by sussygamedeveloper** [reply]
Thanks, i really hate crosh, and crosh hates me lol

**Reply by hello-smile6** [reply]
@sussygamedeveloper Try `help_advanced`

**Reply by sussygamedeveloper** [reply]
ok, will do!

**Reply by BobStadler** [reply]
how can i save it to my local sotrage? it allways ties to go to a external device.
